packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -231,13 +241,8 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         // disputed relay. Because of this, the getState call doesn't impact the gas usage in the happy path.
         require(
             relays[depositHash].relayState == RelayState.Uninitialized ||
-                _getOptimisticOracle().getState(
-                    address(this),
-                    bridgeAdmin.identifier(),
-                    relays[depositHash].priceRequestTime,
-                    _getRelayAncillaryData(_getRelayHash(depositData, relays[depositHash]))
-                ) ==
-                OptimisticOracleInterface.State.Disputed
+                relays[depositHash].relayState == RelayState.Disputed,
```

Comment:
can you explain why we dont need to check the disputed state? it's due to the `relayRequestAncillaryData`  I'm pretty sure but a comment would be a good idea.

Reply:
So this check allows a disputed relay to be "re-relayed" I explain that above. Did you mean something else?

Reply:
no your explanation is good! I was more hoping for a comment in the code to this effect.